### PR TITLE
Adding a cow.

### PR DESCRIPTION
**Pull request by dolphin42**

### CODE REVIEW

File: cow.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+print("""
```
Comment (whale42) [inline cow.py:1]: I like the cow.

**Review by whale42 — COMMENTED** [review]
@shark9000 Take a look too please.

File: cow.py (hunk 1 above)
Comment (whale42) [reply cow.py:1]: @shark9000 try now.

Comment (shark9000) [reply cow.py:1]: @whale42 I like this too.

**Review by whale42 — APPROVED** [review]
Its all set!

File: cow.py (hunk 1 above)
Comment (dolphin42) [reply cow.py:1]: @whale42 Cool. I'm glad you like it.

Comment (dolphin42) [reply cow.py:1]: @whale42

**Pull request merged**